src/ripple/app/main/Application.cpp

```diff
@@ -368,7 +370,10 @@ class ApplicationImp
     std::vector <std::unique_ptr<Stoppable>> websocketServers_;
 
     boost::asio::signal_set m_signals;
-    beast::WaitableEvent m_stop;
+
+    std::condition_variable cv_;
+    std::mutex mut_;
+    bool time_to_stop = false;
```

Comment:
nit: file consistency consideration: `timeToStop_`

Reply:
Consider renaming to `stop_` or something that doesn't infer a `std::chrono::time_point`.

Reply:
Renaming to `isTimeToStop`.